doc/usage.md

```diff
@@ -0,0 +1,22 @@
+# Picoquic CLI Usage
+On Linux after building, the following executables are available in the project root directory. All support "-h" argument to get the detailed usage:
+
+* picohttp_ct: runs various HTTP tests
+* picoquic_ct: runs various QUIC tests
```

Comment:
This is the test program for the HTTP implementation that is part of the distribution. With no arguments, it runs all the HTTP tests.

Reply:
ok. I was not sure what else to say. so did not do anything on the commit.